src/views/SharedSubfolders.vue

```diff
@@ -0,0 +1,185 @@
+<!--
+  - @copyright Copyright (c) 2019 John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @author John Molakvoæ <skjnldsv@protonmail.com>
```

Comment:
Impersonation ? ^^

Reply:
this needs to go, I copied it from an app to get all the boilerplates related to vue